.nvmrc

```diff
@@ -0,0 +1 @@
+11.2.0
```

Comment:
this is very out of date.  Use `14.17.0` for the current LTS version, or `16.3.0` for the latest version, or leave it out if you don't care.  Node 15+ uses npm7+ which changes the `package-lock.json` format slightly.